test/unit/SpinViewer/SpinViewer.spec.js

```diff
@@ -184,15 +194,17 @@ describe("SpinViewer", function() {
 			}, 30);
 		});
 	});
-	describe("#spinTo", function() {
-		var target
+	describe("#spinTo", () => {
+		let target;
 		let inst;
+
 		beforeEach(() => {
 			target = sandbox();
 			target.innerHTML = `<div"></div>`;
```

Comment:
@WoodNeck 

Is it a typo?

Reply:
That looks like a typo for sure. I'll remove them.